hash/bucketer.go

```diff
@@ -99,6 +100,14 @@ func (b *BucketSet) Owner(key string) string {
 	return ret
 }
 
+// BucketList returns the bucket names of this BucketSet in random order.
+func (b *BucketSet) BucketList() []string {
```

Comment:
Is this for testing only? Or it will be used for production code as well?

Reply:
In production code to build websocket connections.